src/test/java/io/strimzi/utils/deployment/installation/InstallationMethod.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.utils.deployment.installation;
+
+public class InstallationMethod {
+    private String namespaceName;
```

Comment:
Do we actually need this as NAMESPACE is part of Constants?

Reply:
I wanted to have it modifiable, but yeah it could be changed when it will be needed, thanks :)